.github/dependabot.yml

```diff
@@ -6,8 +6,15 @@
 version: 2
 updates:
   - package-ecosystem: "gradle" # See documentation for possible values
-    directory: "/examples" # Location of package manifests
+    directory: "/examples/cache" # Location of package manifests
     schedule:
       interval: "daily"
     allow:
       - dependency-name: "software.momento.java:sdk"
+  - package-ecosystem: "gradle" # See documentation for possible values
+    directory: "/examples/cache-with-aws" # Location of package manifests
+    schedule:
+      interval: "daily"
+    allow:
+      - dependency-name: "software.momento.java:sdk"
+      - dependency-name: "software.amazon.awssdk:secretsmanager"
```

Comment:
I'm not 100% if this will be on a new line and couldn't find examples or reference to check.

Reply:
for now let's just only do the momento dep. the purpose here is to test our released packages, we can upgrade other deps more intentionally over time, we don't necessarily need them auto-updated.